### PR TITLE
Markdown Report

### PR DESCRIPTION
**Pull request by EWhite613**
#minor#

### CODE REVIEW

**Comment by ember-cli-visual-acceptance** [comment]
# Visual Acceptance tests:
## Passed: 
### Addition Information: 
 
## Passed: 
### Addition Information:

File: vendor/VisualAcceptance.js [hunk 1]
```diff
@@ -119,7 +123,11 @@ function capture(imageName, width, height, misMatchPercentageMargin, assert) {
               }
             })
             node.innerHTML = '<div class="test fail"> <div class="list-name">  Failed: '+ imageName+' </div> <div class="additional-info"> Addition Information: </div> <div class="images"> <div class="image"> <img class="diff" src="'+data.getImageDataUrl()+'" /> <div class="caption">  Diff   </div> </div> <div class="image">  <img class="input" src="'+image+'" /> <div class="caption"> Current  </div> </div> <div class="image"> <img class="passed" src="'+res.image+'" /> <div class="caption"> Baseline   </div> </div> </div> </div>'
-          }
+            markdown = '## Failed: \n#### Addition Information: \n <table>'
```
Comment (sglanzer) [inline vendor/VisualAcceptance.js]: Failed -> Changed

File: vendor/VisualAcceptance.js [hunk 2]
```diff
@@ -107,7 +110,8 @@ function capture(imageName, width, height, misMatchPercentageMargin, assert) {
             })
             result = true
             node.innerHTML = '<div class="test pass"> <div class="list-name">  Passed: ' + imageName + '</div> <div class="additional-info"> Addition Information: </div> <img src="'+ image + '" /> </div>'
-          } else {
+            markdown = '## Passed: \n#### Addition Information: \n <img>\n'
```
Comment (sglanzer) [inline vendor/VisualAcceptance.js]: I wouldn't even bother displaying Passed, but I do want to see New

File: vendor/VisualAcceptance.js [hunk 3]
```diff
@@ -77,12 +78,14 @@ function capture(imageName, width, height, misMatchPercentageMargin, assert) {
       $(document.getElementById('ember-testing')).removeAttr('style')
       $(document.getElementById('ember-testing-container')).removeAttr('style')
       node.innerHTML = '<div class="test pass"> <div class="list-name"> No passed image. Saving current as baseline: ' + imageName + '</div> <div class="additional-info"> Addition Information: </div> <img src="'+ image + '" /> </div>'
+      markdown = '## No passed image. Saving current as baseline: \n#### Addition Information: \n <img>\n'
```
Comment (sglanzer) [inline vendor/VisualAcceptance.js]: New

**Comment by sglanzer** [comment]
The aspect ratio still looks off on these, I assume that's intended as a followup PR?

**Comment by ember-cli-visual-acceptance** [comment]
# Visual Acceptance Tests
## No changes

**Comment by ember-cli-visual-acceptance** [comment]
# Visual Acceptance tests:
## New
### No new image. Saving current as baseline: 
#### Addition Information: 
 
### No new image. Saving current as baseline: 
#### Addition Information: 
 
### No new image. Saving current as baseline: 
#### Addition Information:

**Comment by sglanzer** [comment]
This is good, we just need a few tweaks to the report titles and is there padding on the top of the svg that pushes it off the screen?

File: index.js [hunk 4]
```diff
@@ -101,9 +101,9 @@ function appendToReport (req, res, options) {
       imgurLinks.push(uploadToImgur(req.body.images[i]))
     }
     if (req.body.type === 'New') {
-      markdownReport.new += '### No new image. Saving current as baseline: ' + req.body.name + '\n#### Addition Information: \n <img src="' + imgurLinks[0] + '" height="160">\n'
+      markdownReport.new += '\n#### Addition Information: \n##### Saving current as baseline: ' + req.body.name + '\n <img src="' + imgurLinks[0] + '" height="160">\n'
```
Comment (sglanzer) [inline index.js]: Can you make this just `\n#### ' + req.body.name + '\n <img...`

**Comment by sglanzer** [comment]
Approved - thanks for all the refinement passes

**Pull request merged**